src/pipelines.js

```diff
@@ -329,7 +434,7 @@ class ZeroShotClassificationPipeline extends Pipeline {
 
         // Insert labels into hypothesis template
         let hypotheses = candidate_labels.map(
-            x => hypothesis_template.replace('{}', x)
+            (            /** @type {string} */ x) => hypothesis_template.replace('{}', x)
```

Comment:
Same here, just writing the types over the variables like `/** @type {string[]} */` helps to infer the type automatically helps to remove these type hints